packages/core/src/components/entity-title/entityTitle.tsx

```diff
@@ -177,7 +177,7 @@ EntityTitle.displayName = `${DISPLAYNAME_PREFIX}.EntityTitle`;
 /**
  * Construct header class name from H{*}. Returns `undefined` if `heading` is not a Blueprint heading.
  */
-function getClassNameFromHeading(heading: React.FC<unknown>) {
+function getClassNameFromHeading(heading: React.FC<any>) {
```

Comment:
It can be valuable to use `unknown` rather than `any` since the former is more future-proof. If someone refactors this function, they might not realize there's some type safety loss with the argument.

Does `() => React.ReactNode` work as an alternative to `React.FC<unknown>`?

Reply:
Ack, will try this out!

Reply:
Changing to `() => React.ReactNode` does not seem to work:
```
Argument of type 'FC<any> | FC<TextProps>' is not assignable to parameter of type '() => ReactNode'.
  Type 'FC<any>' is not assignable to type '() => ReactNode'.
    Target signature provides too few arguments. Expected 1 or more, but got 0.ts(2345)
```
but I do think this leads us to a better solution to use `FC<TextProps>` instead. This function is only used internally in EntityTitle.
```suggestion
function getClassNameFromHeading(heading: React.FC<TextProps>) {
```

Reply:
f50354994ce29779ce51af89b0d30f87d4b527de

Reply:
Nice!